apps/site/assets/ts/stop/components/ExternalMapLink.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactElement } from "react";
+import renderFa from "../../helpers/render-fa";
+import { isIPhone } from "../../hooks/useMobileDetect";
+
+// This is temporary to test out an idea
+const getPlaceID = (stationName: string) => {
```

Comment:
Might want to add the return type to this function or disable eslint for this line.